packages/ffe-datepicker/less/calendar.less

```diff
@@ -157,3 +151,250 @@
         }
     }
 }
+
+.ffe-calendar__header {
+    @margin-top: 0;
+    @margin-bottom: 5px;
+    @margin-left: 0;
+    @margin-right: 0;
+    @margin-between: 13px;
+
+    @breakpoint-1: 520px;
+    @breakpoint-2: 650px;
```

Comment:
Why custom breakpoints?

Reply:
It just seemed right for this component.